### PR TITLE
Deployment updated to remove 'localhost:8080'

### PR DESCRIPTION
**Pull request by donuty**

### CODE REVIEW

File: client/src/components/relatedItems/RelatedItemsWidget.jsx [hunk 1]
```diff
@@ -37,25 +43,72 @@ class RelatedItemsWidget extends Component {
...
+      newArray.push(setCards(item))
+    })
+
+    return newArray;
+  }
+
+  setCards(eachItem) {
+
+    const endpoints = [
+      `http://localhost:8080/styles/${eachItem}`,
```
Comment (DarianAltF4) [inline client/src/components/relatedItems/RelatedItemsWidget.jsx:97]: Change this localhost url as well then you will be good.

**Review by DarianAltF4 — CHANGES_REQUESTED** [review]
Missed 1 more localhost url.

**Review by DarianAltF4 — APPROVED** [review]
Looks great! 😄

File: client/src/components/relatedItems/RelatedItemsWidget.jsx (hunk 1 above)
Comment (donuty) [reply client/src/components/relatedItems/RelatedItemsWidget.jsx]: Thank you!  I just pushed the new update with this localhost edited out.

**Pull request merged**